### PR TITLE
Extend Lit conversion docs

### PR DESCRIPTION
**Pull request by rasbt**
This adds a short section to explain the steps needed when one wants to use the original evaluation harness.

### CODE REVIEW

File: tutorials/convert_lit_models.md [hunk 1]
```diff
@@ -115,5 +115,36 @@ import torch
...
+
+1. Follow the instructions above to load the model into a Hugging Face transformers model.
+
+2. Create a `model.safetensor` file:
+
+```python
+model.save_pretrained("out/hf-tinyllama/converted/")
+```
+
+3. Copy the tokenizer files into the model-containing directory:
```
Comment (awaelchli) [inline tutorials/convert_lit_models.md:136]: Hmm, I was missing this step. This probably explains why I was never successful just converting the checkpoint only (and getting cryptic error). Dang

**Review by awaelchli — APPROVED** [review]

File: tutorials/convert_lit_models.md (hunk 1 above)
Comment (rasbt) [reply tutorials/convert_lit_models.md:136]: Hm, yeah, I think the HF files needed contain more than just the state dict.

File: tutorials/convert_lit_models.md [hunk 2]
```diff
@@ -115,5 +115,36 @@ import torch
 from transformers import AutoModel
 
 state_dict = torch.load('out/hf-tinyllama/converted/model.pth')
-model = AutoModel.from_pretrained("TinyLlama/TinyLlama-1.1B-intermediate-step-1431k-3T", state_dict=state_dict)
+model = AutoModel.from_pretrained("checkpoints/TinyLlama/TinyLlama-1.1B-intermediate-step-1431k-3T", state_dict=state_dict)
```
Comment (carmocca) [inline tutorials/convert_lit_models.md:118]: Is this change right? Isn't this meant to be the repo id?

File: tutorials/convert_lit_models.md [hunk 3]
```diff
@@ -115,5 +115,36 @@ import torch
...
+2. Create a `model.safetensor` file:
+
+```python
+model.save_pretrained("out/hf-tinyllama/converted/")
+```
+
+3. Copy the tokenizer files into the model-containing directory:
+
+```bash
+!cp checkpoints/$repo_id/tokenizer* out/hf-tinyllama/converted
```
Comment (carmocca) [inline tutorials/convert_lit_models.md:139]: This notation is for notebooks. We shouldnt' have it here

```suggestion
cp checkpoints/$repo_id/tokenizer* out/hf-tinyllama/converted
```

File: tutorials/convert_lit_models.md [hunk 4]
```diff
@@ -115,5 +115,36 @@ import torch
...
+```
+
+4. Run the evaluation harness, for example:
+
+```bash
+!lm_eval --model hf \
```
Comment (carmocca) [inline tutorials/convert_lit_models.md:145]:
```suggestion
lm_eval --model hf \
```

**Review by carmocca — APPROVED** [review]

File: tutorials/convert_lit_models.md (hunk 2 above)
Comment (rasbt) [reply tutorials/convert_lit_models.md:118]: Yes, thanks

Comment (carmocca) [reply tutorials/convert_lit_models.md:118]: I think you forgot to push the update

```suggestion
model = AutoModel.from_pretrained("TinyLlama/TinyLlama-1.1B-intermediate-step-1431k-3T", state_dict=state_dict)
```

Comment (rasbt) [reply tutorials/convert_lit_models.md]: Hm weird, I remember pressing the "commit suggestion" button. Maybe I accidentally overwrote it with my local version afterwards. Should be fixed now though

**Pull request merged**